portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/IdentityStoreClientServiceImpl.java

```diff
@@ -57,6 +63,12 @@
 public class IdentityStoreClientServiceImpl implements IdentityStoreClientService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IdentityStoreClientServiceImpl.class);
+    private static final String USERNAME_CLAIM = "http://wso2.org/claims/username";
+    private static final String GROUPNAME_CLAIM = "http://wso2.org/claims/groupname";
```

Comment:
Just checked the identity-mgt test cases. `http://wso2.org/claims/groupName` is the one being used. Better if we can use the same claim everywhere. Either change this or change the places where `groupName` is being used. 

In the domain-config we have used both styles. SCIM uses `givenName` while OIDC uses `given_name`. I'm +1 for using camelcase.